File: TwentyFortyEight/TFELabel.m
//
//  TFELabel.m
//  TwentyFortyEight
//
//  Created by Joshua Caswell on 9/30/16.
//  Copyright © 2016 Josh Caswell. All rights reserved.
//

#import "TFELabel.h"

@implementation TFELabel

#if !TARGET_OS_IPHONE
- (NSString *)text
{
    return [super stringValue];
}

- (void)setText:(NSString *)text
{
    [super setStringValue:text];
}
#endif

@end

File: Common/Cross-Platform Wrappers/TFELabel.swift
//
//  TFELabel.swift
//  TwentyFortyEight
//
//  Created by Joshua Caswell on 10/1/16.
//  Copyright © 2016 Josh Caswell. All rights reserved.
//

#if os(OSX)

import Cocoa

class TFELabel : NSTextField
{
    var text: String {
        
        get {
            return self.stringValue
        }
        
        set {
            self.stringValue = newValue
        }
    }
}

#else
    
import UIKit
    
class TFELabel : UILabel {}
    
#endif
